content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -147,8 +154,27 @@ spec:
 This pod runs in the `Guaranteed` QoS class because `requests` are equal to `limits`.
 
 Topology Manager would consider this Pod. The Topology Manager consults the CPU Manager `static` policy, which returns the topology of available CPUs. 
-Topology Manager also consults Device Manager to discover the topology of available devices for example.com/device.
+Topology Manager also consults the Device Manager to discover the topology of available devices for example.com/device.
 
 Topology Manager will use this information to store the best Topology for this container. In the case of this Pod, CPU and Device Manager will use this stored information at the resource allocation stage.
 
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      limits:
+        example.com/deviceA: "1"
+        example.com/deviceB: "1"
+      requests:
+        example.com/deviceA: "1"
+        example.com/deviceB: "1"
+```
+This pod runs in the `BestEffort` QoS class because there are not CPU and memory requests.
+
+Topology Manager would consider this Pod. The Topology Manager consults the Device Manager to discover the topology of the available devices for example.com/deviceA and example.com/deviceB.
+
+As above Topology Manager will use this information to store the best Topology for this container. Device Manager will then use this when assigning devices to the Pod.
+
 {{% /capture %}}

```

Comment:
We should probably add a section about known issues.

Reply:
Added the ones I could remember off hand, let me know what others I have missed